### PR TITLE
china mobile production practice

### PR DESCRIPTION
**Pull request by jefree-cat**
Signed-off-by: jefree-cat <jiahang_xu@cmbchina.com>

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ This page contains a list of organizations who are using KubeVela in production
...
 | [铭师堂](https://www.mistong.com/)                                                              | ![testing](https://img.shields.io/badge/-development%20&%20testing-green?style=flat) |                                                                                                                                                                                                                                                                                             |
-| China Mobile						                                                              | ![testing](https://img.shields.io/badge/-development%20&%20testing-green?style=flat) |                                                                                                       
+| China Mobile						                                                              | ![production](https://img.shields.io/badge/-production-blue?style=flat) |
```
Comment (wonderflow) [inline ADOPTERS.md:54]: Move to upside align with other production user

Comment (jefree-cat) [reply ADOPTERS.md]: if table sequence != adoption timeline, I will align with other production user.
thanks.

File: ADOPTERS.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ This page contains a list of organizations who are using KubeVela in production
...
 | [Baidu](https://www.baidu.com/)                                                                 | ![production](https://img.shields.io/badge/-production-blue?style=flat)              |                                                                                                                                                                                                                                                                                             |
+| China Mobile						                                                              | ![production](https://img.shields.io/badge/-production-blue?style=flat) 			 |                                                                                                       																																													   |
```
Comment (wonderflow) [inline ADOPTERS.md:37]: Please add a link to China mobile

Comment (nuclearwu) [reply ADOPTERS.md:37]: [移动云] https://ecloud.10086.cn/      development & testing
@jefree-cat @wonderflow

Comment (wonderflow) [reply ADOPTERS.md:37]: please raise a new PR @nuclearwu ， we can close this one

Comment (nuclearwu) [reply ADOPTERS.md:37]: > please raise a new PR @nuclearwu ， we can close this one

ok, done

**Pull request closed without merging**